src/SCRIPTS/BF/HORUS/vtx.lua

```diff
@@ -45,19 +49,150 @@ return {
         { 5658, 5695, 5732, 5769, 5806, 5843, 5880, 5917 }, -- RaceBand
     },
     postLoad = function (self)
-        if self.values[2] == 0 or self.values[3] == 0 or self.values[4] == 0 then
+        if (self.values[2] or 0) < 0 or (self.values[3] or 0) == 0 or (self.values[4] or 0) == 0 then
             self.values = {}
+        else
+            prevBandVal = 0                      -- reset value trackers
```

Comment:
Scoping should be addressed here  These are specified as members but are being stored in the global area.

Reply:
There are many other references below that should probably also be addressed.